pkg/rbd/rbd_util.go

```diff
@@ -177,8 +177,8 @@ func deleteImage(pOpts *rbdVolume, cr *util.Credentials) error {
 		return fmt.Errorf("rbd %s is still being used", image)
 	}
 
-	klog.V(4).Infof("rbd: rm %s using mon %s, pool %s", image, pOpts.Monitors, pOpts.Pool)
-	args := []string{"rm", image, "--pool", pOpts.Pool, "--id", cr.ID, "-m", pOpts.Monitors,
+	klog.V(4).Infof("rbd: trash mv %s using mon %s, pool %s", image, pOpts.Monitors, pOpts.Pool)
+	args := []string{"trash", "mv", image, "--pool", pOpts.Pool, "--id", cr.ID, "-m", pOpts.Monitors,
```

Comment:
This will need to be combined w/ the "ceph rbd trash remove" call otherwise we will just be leaking images to the trash.

Reply:
@dillaman as you know currently we are doing `rbd task add remove` to delete the image from backend

as with this implementation do I need to replace  `rbd task add remove` with `rbd trash mv` and `rbd task add trash remove` to delete image from backend?

if the user does not have the manager permission is `rbd trash mv` enough?

Reply:
I think in a perfect world if `rbd task add trash remove` fails the CSI should rollback and fail the CO request. Otherwise, you are just leaking images into the trash and if folks don't know to go looking in the trash, they will just cause a hidden problem down the road.

Reply:
You would need to delete the clone and run `rbd trash rm`